### DISCUSSION TITLE
Virtual Work lecture 2

### DISCUSSION THREAD

**Discussion by Amir-M-Vahedi** [cooperrc/advanced-dynamics#34] [Logistics]
Hello everyone,

I think in Virtual Work lecture 2 there is a problem with one of the formulas.
<img width="346" alt="image" src="https://github.com/cooperrc/advanced-dynamics/assets/115154998/6dd04caf-0316-482e-9547-bb2b870049ea">
I think it should be like the below form:
$\delta W = 10 \delta x + 20 (\delta y/ \delta x) * \delta x$

**Comment by RebeccaVillanueva** [comment]
Wouldn't what you wrote just cancel out the $\delta x$ in your second term?

**Reply by Amir-M-Vahedi** [reply]
You're right because we multiplied and divided the $\delta y$ by $\delta x$.
But as we find the expression for $\delta y/ \delta x$, we will substitute it in the equation to have a single differential variable for virtula work.
